ui/v2.5/src/components/Tags/TagDetails/Tag.tsx

```diff
@@ -166,9 +166,12 @@ export const Tag: React.FC = () => {
 
   async function onAutoTag() {
     if (!tag?.id) return;
+    if (!tag?.allow_autotag) {
+      Toast.error({content: intl.formatMessage({ id: "toast.autotag_not_allowed" })})
+    }
     try {
       await mutateMetadataAutoTag({ tags: [tag.id] });
-      Toast.success({ content: "Started auto tagging" });
+      Toast.success({content: intl.formatMessage({ id: "toast.started_auto_tagging" })});
```

Comment:
Beat me to it https://github.com/stashapp/stash/pull/1927/files#diff-74e92961bf92654d420b28e6c909729ee5c396b44af0eaa9b3b44c8137bbf526R24